app/models/katello/content_view_version_export_history.rb

```diff
@@ -20,5 +20,11 @@ class ContentViewVersionExportHistory < Katello::Model
     scoped_search :on => :content_view_id, :relation => :content_view_version, :validator => ScopedSearch::Validators::INTEGER, :only_explicit => true
     scoped_search :on => :content_view_version_id, :only_explicit => true, :validator => ScopedSearch::Validators::INTEGER
     scoped_search :on => :id, :only_explicit => true, :validator => ScopedSearch::Validators::INTEGER
+
+    def self.pick_recent_history(content_view, destination_server: nil)
```

Comment:
What do you think about something like:

```
def self.latest(...)
  where(content_view: :foo, destination_server: :bar).order(:created_at).first
end
```
Is there an advantage to your approach I'm not seeing?

Reply:
I was trying to make sure it wouldnt load every thing and pick the last.. But I think you approach looks better. Going to give it a try